owrx/wsjt.py

```diff
@@ -197,6 +209,25 @@ def getMode(self):
         return "FST4W"
 
 
+class JT4Profile(WsjtProfile):
+    availableSubmodes = ["A", "B", "C", "D", "E", "F", "G"]
+
+    def __init__(self, submode):
+        self.submode = submode
+
+    def getInterval(self):
+        return 60
+
+    def getSubmode(self):
```

Comment:
I don't see this method being used anywhere. Any plans for this? If not, I'd recommend removing it.

Reply:
Correcred.